pkg/fetch/bellsoft/hardened-containers/osv/types.go

```diff
@@ -0,0 +1,51 @@
+package osv
+
+type OSV struct {
+	SchemaVersion string   `json:"schema_version,omitempty"`
+	ID            string   `json:"id,omitempty"`
+	Modified      string   `json:"modified,omitempty"`
+	Published     string   `json:"published,omitempty"`
+	WithDrawn     string   `json:"with_drawn,omitempty"`
```

Comment:
```suggestion
	Withdrawn     string   `json:"withdrawn,omitempty"`
```

"Withdraw" is a single word, is it better to not use underscore?
In OSV format, there is "withdrawn" field, too: https://ossf.github.io/osv-schema/#withdrawn-field

Reply:
I plan to fix it all together later.